src/telemetry/metadata.c

```diff
@@ -1,108 +0,0 @@
-/*
```

Comment:
It is still not clear to me why this file is deleted rather than renamed like other files. Not sure why it needs renaming either, given that it is namespaced by the telemetry dir already. In any case, I'd like to see this file moved rather than deleted if possible because I believe it helps rebasing.

Reply:
Git just seems incapable of recognizing a move if the file is sufficiently modified with the same change.  Nothing I try here seems to make any difference.  I split the move and modifications into two changes, both of which looked correct when view by themselves, but when viewing a PR of the pair of changes, it still presented it as a delete and new file.